app/controllers/admin/bulk_assign_role_controller.rb

```diff
@@ -38,13 +38,11 @@ def assign_role
             },
           )
         end
-
         flash[:success] = I18n.t("admin.bulk_assign_role_controller.success_message")
-        redirect_to admin_bulk_assign_role_index_path
       rescue StandardError => e
         flash[:danger] = e.message
-        redirect_to admin_bulk_assign_role_index_path
       end
+      redirect_to admin_bulk_assign_role_index_path
     rescue ArgumentError => e
       flash[:danger] = e.message
       redirect_to admin_bulk_assign_role_index_path
```

Comment:
Ah, I see: the repeating `redirect_to` is because of the `rescue`. I think we could get it to one `redirect_to` with an `ensure`, but it's not necessary to be quite so minimalist.